File: src/main/java/com/intellij/tapestry/psi/TmlElementType.java
package com.intellij.tapestry.psi;

import com.intellij.psi.tree.IFileElementType;
import com.intellij.tapestry.lang.TmlLanguage;

/**
 * @author Alexey Chmutov
 */
public interface TmlElementType {
  IFileElementType TML_FILE = new IFileElementType("TML_FILE", TmlLanguage.INSTANCE);
}

File: src/main/kotlin/com/github/rar91279/plugin/tapestry/psi/TmlElementType.kt
package com.github.rar91279.plugin.tapestry.psi

import com.intellij.psi.tree.IFileElementType
import com.github.rar91279.plugin.tapestry.lang.TmlLanguage

/** The file element type of a Tapestry template. */
object TmlElementType {

    @JvmField
    val TML_FILE: IFileElementType = IFileElementType("TML_FILE", TmlLanguage)
}
